less/_grid.less

```diff
@@ -76,9 +76,13 @@
 
 // other CSS for the container of the new columns
 .grid {
+	@grid-sides: 15px;
 	.clearfix;
 
-	width: 100%;
+	padding: 0 25px 40px 25px;
```

Comment:
I am having some thoughts on this padding in the `.grid`  I'll see if I can't work them out when I get to the office.